_hooks/pre-commit

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# A git hook script that runs the same checks that are executed on Travis to
+# verify your commits will not fail. Bypass it with the --no-verify option to
+# git commit
+#
+# usage: make a symlink to this file,
+# e.g., rm -rf .git/hooks/ && ln -s -f ../_hooks .git/hooks
+
+#######################################
+# Error function to display error message to end users.
+# Arguments:
+#   Error message description
+#######################################
+err() {
+  echo "$@"
+  exit 1
+}
```

Comment:
Oh sorry, I actually meant we should wholesale steal the example from here:
https://google.github.io/styleguide/shell.xml?showone=STDOUT_vs_STDERR#STDOUT_vs_STDERR

Primarily because it prints to stderr instead of stdout. Also I'd prefer not to bake the "exit" into the function.